src/main/java/com/ericsson/ei/controller/FailedNotificationsControllerImpl.java

```diff
@@ -56,25 +54,26 @@
      * @param subscriptionName
      */
     @Override
-    @ApiOperation(value = "Retrieve missed notifications", response = QueryResponse.class)
-    public ResponseEntity<?> getQueryMissedNotifications(@RequestParam("SubscriptionName") final String subscriptionName) {
+    @ApiOperation(value = "Retrieve failed notifications", response = QueryResponse.class)
+    public ResponseEntity<?> getFailedNotifications(@RequestParam("SubscriptionName") final String subscriptionNames) {
         ObjectMapper mapper = new ObjectMapper();
         QueryResponse queryResponse = new QueryResponse();
         QueryResponseEntity queryResponseEntity = new QueryResponseEntity();
         try {
-            List<String> response = processMissedNotification.processQueryMissedNotification(subscriptionName);
+            List<String> response = processMissedNotification.processQueryMissedNotification(subscriptionNames);
             if (!response.isEmpty()) {
                 queryResponseEntity = mapper.readValue(response.get(0), QueryResponseEntity.class);
             }
             queryResponse.setQueryResponseEntity(queryResponseEntity);
             LOGGER.debug("The response is : {}", response.toString());
-            if (processMissedNotification.deleteMissedNotification(subscriptionName)) {
-                LOGGER.debug("Missed notification with subscription name {} was successfully removed from database", subscriptionName);
+            if (processMissedNotification.deleteMissedNotification(subscriptionNames)) {
+                LOGGER.debug("Failed notification for subscription {} was successfully removed from database",
+                        subscriptionNames);
             }
             return new ResponseEntity<>(queryResponse, HttpStatus.OK);
         } catch (Exception e) {
-            String errorMessage = "Internal Server Error: Failed to extract the data from the Missed Notification Object based on subscription name "
-                    + subscriptionName + ".";
+            String errorMessage = "Failed to extract the data from the Failed Notification Object based on subscription name "
```

Comment:
"Failed Notification Object" should be lower-cased, it is not a name.

Reply:
done